todo_app/data/trello_items.py

```diff
@@ -18,209 +29,203 @@ def __init__(
         date,
         dueDate=None,
         description=None,
-        status="To Do",
+        listName="To Do",
     ):
         self.id = id
         self.name = name
-        self.status = status
+        self.listName = listName
         self.date = date
         self.dueDate = dueDate if dueDate is not None else "12-12-12"
         self.description = (
             description if description is not (None or "") else "No description"
         )
 
     @classmethod
-    def from_trello_card(cls, card, list):
-        lastActiveDate = datetime.strptime(
-            card["dateLastActivity"], "%Y-%m-%dT%H:%M:%S.%fZ"
-        ).strftime("%d/%m/%y")
+    def from_trello_card(cls, card):
+        lastActiveDate = card["dateLastActivity"].strftime("%d/%m/%y")
         if card.get("due"):
-            dueDate = datetime.strptime(card["due"], "%Y-%m-%dT%H:%M:%S.%fZ").strftime(
+            dueDate = card["due"].strftime(
                 "%d/%m/%y"
             )
         else:
             dueDate = None
+        
+        listName = "To Do"
+        #get list name
+        if card["listId"] == "657c834ff1b03a6ddfeadc94":
+            listName = "Done"
+        elif card["listId"] == "657c834ff1b03a6ddfeadc93":
+            listName = "Doing"
+
 
         return cls(
-            card["id"],
+            card["_id"],
             card["name"],
             lastActiveDate,
             dueDate,
             card["desc"],
-            list["name"],
-        )
-
-
-def get_query():
-    return {
-        "key": os.environ.get("TRELLO_KEY"),
-        "token": os.environ.get("TRELLO_TOKEN"),
-    }
-
-
-# get boards avaialble to the user
-def get_boards():
-    searchUrl = baseUrl + "members/me/boards?fields=name"
-    try:
-        response = requests.request(
-            "GET", searchUrl, headers=headers, params=get_query()
+            listName,
         )
-        response.raise_for_status()
-        response = response.json()
-    except requests.exceptions.RequestException as exception:
-        print(f"An error occured: {exception}")
-        response = None
-    return response
 
 
-# get the cards in a list
-def get_list_cards(id):
-    searchUrl = baseUrl + f"lists/{id}/cards"
-
-    try:
-        response = requests.request(
-            "GET", searchUrl, headers=headers, params=get_query()
-        )
-        response.raise_for_status()
-        response = response.json()
-    except requests.exceptions.RequestException as exception:
-        print(f"An error occured: {exception}")
-        response = None
-    return response
-
-
-# get the lists on  board
-def get_boardLists(id):
-    searchUrl = f"{baseUrl}boards/{id}/lists"
+# Add a new document to the DBcard
 
+def add_card(listId, cardName, desc, due, board, addedCollection = None):
+    collection = cardsCollection
+    if addedCollection is not None:
+        collection = addedCollection
```

Comment:
These lines feel a bit odd to me, but if I understand rightly they're so that the tests can intercept the cardsCollection essentially? I think that might be handled nicer by patching the cardsCollection directly, so that this code doesn't have to acknowledge that slightly messy step
(Or if handled using the wrapping approach above, this should resolve itself through use of MongoMock)